src/libs/actions/Report.ts

```diff
@@ -1806,7 +1806,7 @@ function navigateToConciergeChatAndDeleteReport(reportID: string) {
 /**
  * @param policyRoomName The updated name for the policy room
  */
-function updatePolicyRoomNameAndNavigate(policyRoomReport: Report, policyRoomName: string) {
+function updatePolicyRoomNameAndNavigate(policyRoomReport: Report, policyRoomName: string | undefined) {
```

Comment:
```suggestion
function updatePolicyRoomNameAndNavigate(policyRoomReport: Report, policyRoomName: string) {
```

This one doesn't need `undefined`